pkgs/applications/audio/drumgizmo/default.nix

```diff
@@ -11,6 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "0bpbkzcr3znbwfdk79c14n5k5hh80iqlk2nc03q95vhimbadk8k7";
   };
 
+  # from: http://cgit.drumgizmo.org/plugingizmo.git/commit/?id=be64ddf9da525cd5c6757464efc966052731ba71
+  patches = [ ./lv2.patch ];
```

Comment:
Is it not sufficient to add `http://cgit.drumgizmo.org/plugingizmo.git/patch/?id=be64ddf9da525cd5c6757464efc966052731ba71` to `patches` using `fetchpatch`?

Reply:
That patch doesn't apply:
```
applying patch /nix/store/wkfr0iviwksigwgr6f3wcr8j3ll2djds-?id=be64ddf9da525cd5c6757464efc966052731ba71
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/pluginlv2.cc
|+++ b/pluginlv2.cc
--------------------------
File to patch:
Skip this patch? [y]
Skipping patch.
2 out of 2 hunks ignored
can't find file to patch at input line 23
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/pluginlv2.h
|+++ b/pluginlv2.h
--------------------------
File to patch:
Skip this patch? [y]
Skipping patch.
2 out of 2 hunks ignored
builder for '/nix/store/pwayx3ps4gg7w4baxslr7jydh1nqs1b1-drumgizmo-0.9.18.1.drv' failed with exit code 1
principal> process result: Terminated 199
```

Reply:
Might need to pass `extraPrefix = "plugin/plugingizmo/"` to `fetchpatch`.

Reply:
Thanks, but that gives:
```
unpacking sources
unpacking source archive /nix/store/c4di77j7lpw50rmjyyqkf33hvyndas74-drumgizmo-0.9.18.1.tar.gz
source root is drumgizmo-0.9.18.1
setting SOURCE_DATE_EPOCH to timestamp 1571848838 of file drumgizmo-0.9.18.1/plugin/manifest.ttl
patching sources
applying patch /nix/store/76r0hwi5168cczwhxlxpz2d7ygvwbhi1-?id=be64ddf9da525cd5c6757464efc966052731ba71
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/plugin/plugingizmo/a/pluginlv2.cc
|+++ b/plugin/plugingizmo/b/pluginlv2.cc
--------------------------
File to patch:
Skip this patch? [y]
Skipping patch.
2 out of 2 hunks ignored
can't find file to patch at input line 23
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/plugin/plugingizmo/a/pluginlv2.h
|+++ b/plugin/plugingizmo/b/pluginlv2.h
--------------------------
File to patch:
Skip this patch? [y]
Skipping patch.
2 out of 2 hunks ignored
```

Reply:
Then also `stripLen = 1;` to remove the original `a/` prefix.

Reply:
Yes, that works.
Thank you!